test/fixtures/hooks/multi/10-print_hello_kafo.rb

```diff
@@ -0,0 +1,9 @@
+boot do
```

Comment:
Other ideas:
* test that you can instantiate multiple hooks in the same file
* test that you can give hooks names which is important for ordering

Reply:
> * test that you can instantiate multiple hooks in the same file

Multiple hooks of the same *type* (e.g. 2 boot hooks) ? Since I technically am testing multiple hooks already :)

Reply:
> * test that you can instantiate multiple hooks in the same file
> * test that you can give hooks names which is important for ordering

I think I hit these in one go by adding an additional `post` hook with names and ordering expressed in the names and then testing the output is in the correct order.

Reply:
> Multiple hooks of the same _type_ (e.g. 2 boot hooks) ? Since I technically am testing multiple hooks already :)

Yes. Technically I think it's supported if you use names but it doesn't exist unless there's tests, right? :)

> I think I hit these in one go by adding an additional `post` hook with names and ordering expressed in the names and then testing the output is in the correct order.

Pretty sure I'd do the same thing.